Potatohead.java

```diff
@@ -82,7 +91,11 @@ public void setHasMouth(boolean _hasMouth){
     }
     // STEP 4 OF 5: a mutator method for yr instance var WITH JAVADOC COMMENT
     // !!! NOTE: You MUST add Javadoc comments for your accessor method and mutator method
-
+    
+	
+    public boolean getHasEars(){
```

Comment:
Line 96: Missing Javadoc comment for public boolean getHasEars()

Please fix and resubmit